docs/src/references/c8y-log-management.md

```diff
@@ -20,21 +20,20 @@ As part of this plugin installation:
 * On systemd enabled devices, the service definition file for this `c8y_log_plugin` daemon is also installed as part of this plugin installation.
 
 Once installed, the `c8y_log_plugin` is run as a daemon on the device listening to log requests from Cumulocity on `c8y/s/us` MQTT topic.
-On startup, it reports all the log file types that it manages, defined in the `c8y_log_plugin.toml`
+On startup, it reports all the log file types that it manages, defined in the `c8y-log-plugin.toml`
 
 ## Configuration
 
-The `c8y_log_plugin` configuration is stored by default under `/etc/tedge/c8y/c8y_log_plugin.toml`.
+The `c8y_log_plugin` configuration is stored by default under `/etc/tedge/c8y/c8y-log-plugin.toml`.
 
 This [TOML](https://toml.io/en/) file defines the list of log files that can be retrieved from the cloud tenant.
 The paths to these files can be represented using [glob](https://en.wikipedia.org/wiki/Glob_(programming)) patterns.
 The `type` given to these paths are used as the log type when they are reported to Cumulocity.
 
 ```toml
 files = [
-    { type = "mosquitto", path = '/var/log/mosquitto.log' },
-    { type = "software-management", path = '/var/log/tedge/agent/software-management/software-*' },
-    { type = "c8y_LogRequest", path = '/var/log/tedge/agent/c8y_LogRequest/*' },
```

Comment:
I was surprised to see that line removed ... but I double checked: these logs are indeed never created.